### PR TITLE
Features/ci setup 176963132

### PR DESCRIPTION
**Pull request by rosalinekarr**
Problem
=======
We want to automatically run the lint script when branches are pushed.
[#176963132](https://www.pivotaltracker.com/story/show/176963132)

Solution
========
Added GitHub Actions workflow for running `rpm run lint`.

Change summary:
---------------
* Added a CI GitHub workflow at `.github/workflows/ci.yml`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: [push]
```
Comment (wilwade) [inline .github/workflows/ci.yml:3]: Do we want to limit the linting to just PRs?

**Review by wilwade — COMMENTED** [review]
One question about the `on: push`, but otherwise looks good. (Also if you update the ubuntu version, also update this one.)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rosalinekarr) [reply .github/workflows/ci.yml]: fixed

**Review by shannonwells — APPROVED** [review]
👍

**Review by wilwade — APPROVED** [review]
Rebase, but otherwise looks good.

**Pull request merged**